exercices/exercice-3/src/App.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div id="app">
+    <div class="container">
+      <div class="cover">
+        <img src="./assets/zenidex.png"/>
+      </div>
+      <div class="m-t">
+        <search v-on:onSearchChange="onSearchChange"/>
```

Comment:
Use the v-on shorten syntax @ instead
`@onSearchChange="onSearchChange" `

Reply:
Was thinking about the readme that made me use `v-on` instead of shorten syntax :p. I push a fix